### DISCUSSION TITLE
Big Pictures saved as JPG instead of PNG

### DISCUSSION THREAD

**Discussion by MrLawbreaker** [vladmandic/sdnext#666] [Q&A] [answered]
When SD creates a big image it saves it as a JPG instead of PNG, any way to change that, I know in A1111 there is an option to toggle this but this seems to be missing here?

**Comment by TKS4** [comment]
There is a setting in the Image Options tab that talks about Maximum Megapixels (default is 200MB I think when Auto's was 4MB if I recall). I think maybe if you change that threshold to something higher, it might fix your issue? Though I'm having a hard time understanding if you're actually hitting this 200mb threshold and the fallback encoding goes to JPEG in an effort to save space. How big are these images just wondering? Maybe PNG's have a max dimension size?

Or were you talking about that somewhat confusing functionality in Auto's that downscales the image?

**Reply by vladmandic** [reply]
the maximum megapixel is hard limit to avoid out-of-memory situations (e.g., if you try to save xyz grid that is so big it would take too much memory), thats not the same.

there was a feature in a1111 that worked like 
- if selected type is png
- if generated png is above xxx, convert to jpg forcefully

that feature is removed. there are a lot of tiny features that may be useful in some borderline cases, but there were thousands of them and were nearly-never used.

**Reply by MrLawbreaker** [reply]
so currently my best bet is to just set JPG quality to 100?

**Reply by vladmandic** [reply, marked as answer]
You can try lossless webp
